### DISCUSSION TITLE
Hinted userscript with command line arguments, possible?

### DISCUSSION THREAD

**Discussion by jcguu95** [qutebrowser/qutebrowser#7341] [Q&A] [answered]
I have the following userscript caller that helps me add youtube links to a file for later downloading. Notice that it uses a single command line argument `audio` and `video`.

```
# It works! :)
config.bind('aa',"spawn --userscript "+os.environ["HOME"]+"/.scripts/qb/ytd.bb audio")
config.bind('av',"spawn --userscript "+os.environ["HOME"]+"/.scripts/qb/ytd.bb video")
```

However, sometimes I'd like to spawn the script with `hint links`. However, the following callers fail because it treats `.../ytd.bb audio` together as the script itself. Is there any workaround?

```
# It doesn't work :(
config.bind(';aa',"hint links userscript '"+os.environ["HOME"]+"/.scripts/qb/ytd.bb audio'") # FIXME
config.bind(';av',"hint links userscript "+os.environ["HOME"]+"/.scripts/qb/ytd.bb video") # FIXME
```

**Comment by The-Compiler** [comment, marked as answer]
See #3400

**Comment by jcguu95** [comment]
I see. So it's currently impossible but we can workaround with a wrapper.
